redisinsight/api/src/modules/workbench/providers/workbench-commands.executor.ts

```diff
@@ -144,7 +157,9 @@ export class WorkbenchCommandsExecutor {
 
       return {
         ...rest,
-        response: this.formatter.format(rest.response),
+        response: mode === WorkbenchMode.Raw
+          ? this.utf8Formatter.format(rest.response)
+          : this.rawFormatter.format(rest.response),
```

Comment:
again here. we should not check mode in this place. It will be better to determine and set formatter somewhere above. this formatter shold implement some interface. so you can write something like this:
```
const formater = { determine formater based on type and return new FormaterUTF8... } // this formater should implement IFormater interface

then below in the code you can simply do formater.format() without care about anything 
```